app/src/test/java/com/devmob/alaya/ui/screen/ContactViewModelTest.kt

```diff
@@ -29,6 +36,49 @@ class ContactViewModelTest {
         assertEquals("+54 9 1123456789", viewModel.formatPhoneNumberForWhatsApp("1123456789"))
     }
 
+//ESTOS TYEST FALLAN, BUSCAR LA FORMA DE TESTEAR LOS INTENT
```

Comment:
Podríamos probar acá la herramienta Robolectric, lo sume en el PR de Voice to Text
Te parece dejarte la tarea de corregir estos tests luego implementando robolectric???